src/configuration/WithTemplateConfig/utils/getArrayFromTemplate.ts

```diff
@@ -6,10 +6,20 @@ export const getArrayFromTemplate = (template?: WithTemplate) => {
   if (!template) {return []}
   const { expr } = template
   const arr = splitByCommaOutsideBrackets(expr)
-  return arr.filter(a => a).map(a => ({
-    label: a.split("=")[0].trim().replace(/\(.+\)/, ""),
-    value: a.trim()
-  }))
+
+  return arr.filter(a => a).map(a => {
+    const commentMatch = a.match(/#.*\n/gm);
+    const comment = commentMatch ? commentMatch.join('').trim() : '';
+
+    const variableMatch = a.match(/(.*?)=/);
+    const variableName = variableMatch ? variableMatch[0].slice(0, -2) : '';
```

Comment:
variableMatch may be an empty the slice and there can be error when we are trying to get 0 element on empty slice